src/services/aws_macros.ts

```diff
@@ -26,9 +26,13 @@ import { SQS } from '@aws-sdk/client-sqs';
 import { SSM } from '@aws-sdk/client-ssm';
 import { STS } from '@aws-sdk/client-sts';
 import { defaultRetryDecider, StandardRetryStrategy } from '@aws-sdk/middleware-retry';
+import { NodeHttpHandler } from '@aws-sdk/node-http-handler';
+import { HttpProxyAgent, HttpsProxyAgent } from 'hpagent';
 
 import config from '../config';
 
+const useProxy = !!process.env.HTTPS_PROXY && !!process.env.HTTP_PROXY;
```

Comment:
Is not possible to be defined just one of them?

Reply:
It is, but I would consider that a misconfiguration (for now, at least). The [AWS SDK Docs](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/node-configuring-proxies.html) imply in the call-out at the end that both are necessary to be sure the SDK works. It's possible that this isn't really true and we can get away with just the `HTTPS_PROXY` instead of both, but I am erring on the safe side for now -- especially since the user who needs this defines both.